handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -313,31 +333,58 @@ public void operationComplete(Future<SslContext> future) throws Exception {
     }
 
     /**
-     * The default implementation will simply call {@link AsyncMapping#map(Object, Promise)} but
-     * users can override this method to implement custom behavior.
+     * The default implementation will simply call {@link SslContextMapping#map(String, Promise)} but users can
```

Comment:
Because `SslContextMapping extends AsyncMapping<String, ...>` it's `SslContextMapping#map(String, Promise)`.